new/LIT6291HassabaRetu.xml

```diff
@@ -0,0 +1,82 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6291HassabaRetu" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ሐሳበ፡ ርቱዕ፡</title>
+                <title type="normalized" xml:lang="gez" corresp="#t1">Ḥassāba rәtuˁ</title>
+                <title xml:lang="en" xml:id="t2">Computus of the Righteous</title>
```

Comment:
I would encode it as a translation of t1, not as a new title

Reply:
yes.